### DISCUSSION TITLE
I suggested Elsa workflow as a workflow component in .NET 5.0

### DISCUSSION THREAD

**Discussion by Alexplose** [elsa-workflows/elsa-core#553] [Show and tell]
Just to let you know that I referenced Elsa in .NET 5.0 docs.
~~https://github.com/Alexplose/docs/blob/patch-1/docs/core/dotnet-five.md~~
https://github.com/dotnet/docs/blob/master/docs/core/dotnet-five.md

**Comment by sfmskywalker** [comment]
Sweet!
PS. The link leads to a 404.

**Reply by Alexplose** [reply]
Sorry can't even post a valid link, here is the right one : 
https://github.com/dotnet/docs/blob/master/docs/core/dotnet-five.md
